packages/keystone/package.json

```diff
@@ -18,9 +18,9 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@web3-onboard/common": "^2.0.7",
     "@ethereumjs/tx": "^3.4.0",
     "@ethersproject/providers": "^5.5.0",
-    "@keystonehq/eth-keyring": "^0.11.2-alpha.2"
+    "@keystonehq/eth-keyring": "^0.14.0-alpha.10.3",
```

Comment:
Remove alpha from here as well?

Reply:
I'm not too sure on this one. @taylorjdawson thoughts?

Reply:
Oh yes actually this is an outside dep. Nevermind!